Sources/libcoverage/include/libcoverage.h

```diff
@@ -20,7 +20,7 @@
 
 struct edge_set {
     uint64_t count;
-    unsigned int* edges;
+    uint64_t * edges;
```

Comment:
The switch to uint64_t was due to an initial concern I had that long fuzzing runs could result in overflow, especially when I was considering having instances share edge counts. I did some rough math, and it looks like it will take would take a single core more than a year to overflow. I'll switch it back.